localization/pose_initializer/src/pose_initializer/ndt_module.cpp

```diff
@@ -40,7 +40,6 @@ PoseWithCovarianceStamped NdtModule::align_pose(const PoseWithCovarianceStamped
   RCLCPP_INFO(logger_, "Call NDT align server.");
   const auto res = cli_align_->async_send_request(req).get();
   if (!res->success) {
-    RCLCPP_INFO(logger_, "NDT align server failed.");
```

Comment:
@ahmeddesokyebrahim Could you please not delete this message?

Reply:
@YamatoAndo 
That is a sure thing, I can revert this change.
The only motivation of deleting it is that if the server failed, that will be logged as error with the `throw ServiceException`.

Reply:
I see. So, the message is duplicated.
However, since it's an important error message, I'd like it to be output not only in the logs but also in the terminal.

Reply:
@YamatoAndo : just fyi, the error message is visible in the terminal as well.

Reply:
@ahmeddesokyebrahim Thank you!
In that case, it's okay to delete the `RCLCPP_INFO` message.